public/scripts/utils.js

```diff
@@ -44,4 +44,99 @@ module.exports = {
       .normalize('NFD')
       .replace(/[\u0300-\u036f]/g, '');
   },
+
+  sortByColors: function (colors) {
```

Comment:
This should not be part of this file, this file contains utility functions for the website when displayed in the browser. This function will not be used when the website is live.

It should probably go in a file in [the `scripts` directory](https://github.com/simple-icons/simple-icons-website/tree/00139387a7133ce1eadd86eb70a0e28e8f35dac4/scripts) ([following the pattern of the main repo](https://github.com/simple-icons/simple-icons/blob/91a945820677dfff652920cac065669b350e194b/scripts/utils.js)).

Reply:
moved to its own file in the scripts folder